fuzzing/libfuzzer/libs/gramine_fuzzing_libs.py

```diff
@@ -0,0 +1,73 @@
+import os
+from libs import utils
+from common.config.constants import *
+import subprocess
+
+
+def build_libfuzzer(filesize):
+    with open(os.path.join(LIBFUZZER_CORPUS_DIR, str(filesize)), 'wb') as file:
+        file.write(os.urandom(filesize))
+    print(utils.exec_shell_cmd('clang -g -fsanitize=fuzzer example.c -o example_fuzzer'))
+    print(utils.exec_shell_cmd('gramine-sgx-pf-crypt encrypt -w files/wrap_key -i corpus -o cipher_corpus'))
+    print(utils.exec_shell_cmd('make clean'))
+    print(utils.exec_shell_cmd('make SGX=1'))
+
+def run_libfuzzer(testname):
+    for i in range(5):
```

Comment:
Take the number of iterations as Jenkins parameter,  if nothing is passed then set the default to 5.

Reply:
added environment variable